var/spack/repos/builtin/packages/magma/package.py

```diff
@@ -36,6 +36,7 @@ class Magma(CMakePackage, CudaPackage):
     depends_on('blas')
     depends_on('lapack')
     depends_on('cuda@8:', when='@2.5.1:')  # See PR #14471
+    depends_on('cuda@:10.2.89')  # Incompatible with CUDA 11
```

Comment:
Potentially better:
```suggestion
    depends_on('cuda@:10.2.89', when='@:2.5.3')  # incompatible with CUDA 11
```

Reply:
@ax3l do we know of any work in the magma project to support cuda11?

Reply:
@becker33 Yes, the MAGMA team is working on CUDA 11 support in the next release.